source/SIL.AppBuilder.Portal/src/routes/(authenticated)/admin/workflows/[product_id]/+page.server.ts

```diff
@@ -0,0 +1,86 @@
+import { prisma, DefaultWorkflow, getSnapshot } from 'sil.appbuilder.portal.common';
+import { transform, type StateName } from 'sil.appbuilder.portal.common/workflow';
+import { createActor, type Snapshot } from 'xstate';
+import type { PageServerLoad, Actions } from './$types';
+import { fail, superValidate } from 'sveltekit-superforms';
+import { valibot } from 'sveltekit-superforms/adapters';
+import * as v from 'valibot';
+
+const jumpStateSchema = v.object({
+  state: v.string()
+});
+
+export const load: PageServerLoad = async ({ params, url, locals }) => {
+  const actor = createActor(DefaultWorkflow, {
```

Comment:
Probably don't need to create actor here. Can just use Value and Context in load.